app/src/common/shared/com/igalia/wolvic/ui/widgets/dialogs/PrivacyPolicyDialogWidget.java

```diff
@@ -0,0 +1,98 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package com.igalia.wolvic.ui.widgets.dialogs;
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.view.LayoutInflater;
+
+import androidx.annotation.NonNull;
+import androidx.databinding.DataBindingUtil;
+
+import com.igalia.wolvic.R;
+import com.igalia.wolvic.databinding.PrivacyPolicyDialogBinding;
+import com.igalia.wolvic.ui.widgets.WidgetPlacement;
+import com.igalia.wolvic.ui.widgets.WindowWidget;
+
+/**
+ * A dialog that displays the Privacy Policy along with two buttons
+ * to accept or reject it.
```

Comment:
Nit: I think this fits nice in one line.

Reply:
Done.